tests/test_wave_01.py

```diff
@@ -54,6 +54,7 @@ def test_letter_not_selected_too_many_times():
     for i in range(1000):
         # Arrange/Act
         letters = draw_letters()
+        print(letters)
```

Comment:
We shouldn't commit changes to files that we don't intend to share out. It's all good to add code for testing or reformat things locally while we're working, but when we do, we should make sure that when we stage file for commit that we specify which files we are adding over using shortcuts that add all changed files. When we go to create a PR, we're also given a chance to review our changes before clicking "open" and that's a great place to double check that you only see the changes you intended to make.